src/plugins/data_source_management/public/components/create_data_source_wizard/create_data_source_wizard.tsx

```diff
@@ -29,14 +29,40 @@ export const CreateDataSourceWizard: React.FunctionComponent<CreateDataSourceWiz
   const toastLifeTimeMs: number = 6000;
 
   /* State Variables */
+  const [existingDatasourceNamesList, setExistingDatasourceNamesList] = useState<string[]>([]);
   const [toasts, setToasts] = useState<EuiGlobalToastListToast[]>([]);
   const [isLoading, setIsLoading] = useState<boolean>(false);
 
   /* Set breadcrumb */
   useEffectOnce(() => {
     setBreadcrumbs(getCreateBreadcrumbs());
+    getExistingDataSourceNames();
   });
 
+  /* fetch datasources */
+  const getExistingDataSourceNames = async () => {
+    setIsLoading(true);
+    try {
+      const listOfDataSources: DataSourceTableItem[] = await getDataSources(savedObjects.client);
+
+      if (Array.isArray(listOfDataSources) && listOfDataSources.length) {
+        setExistingDatasourceNamesList(
+          listOfDataSources.map((datasource) => datasource.title?.toLowerCase())
+        );
+      }
+    } catch (e) {
+      handleDisplayToastMessage({
```

Comment:
Let's use the toast from notification service to add toast. Thanks for @ashwin-pc to point this out.

Reply:
Done